torch_disc/torch_disc/csrc/disc_compiler/disc_compiler.cpp

```diff
@@ -33,13 +34,12 @@ std::vector<torch::lazy::BackendDataPtr> Executable::Run(
     } else {
       // TODO(whc) should this check be made more general? it's written somewhat
       // oddly
-      CHECK(default_device_is_cuda ||
+      CHECK(!default_device_is_cuda ||
```

Comment:
What does this mean?

Reply:
just following the `TSBackendImpl` code, I'm not sure whether to delete them yet.